README.md

```diff
@@ -2,31 +2,311 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Define character encoding as ASCII and Unicode values
+- Discuss escape sequences
+- Compare character array and string representations in memory
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+So far, we have seen the `char` data type  in passing, and we know it
+stores a single character or ASCII value, but we really haven't used it or
+talked too in-depth about it. In this section, we will explore characters some
+more, talk about what an ASCII value is, point out a few special characters,
+and explain how it might relate to the `String` class.
 
-## Topic 1
+## The `char` data type
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+A `char` is a primitive data type that stores a single character or an ASCII value.
+Let's explore what "ASCII value" means.
 
-## Topic 2
+`char grade = 'A';`
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The above code is how we would declare a variable named `grade` to store a `char` value,
+and then assign an initial value by enclosing the character in single quotes, such as 'A'.
+
+Since everything in memory is stored as a binary value, we need to
+convert the letter 'A' into some type of numeric value in order to store it.
+**ASCII** is a standard for encoding characters as decimal values.
+An **ASCII Table** is a tabular representation that links characters to decimal
+values. We can see what each letter represents using the ASCII table below. We can
+also see that the uppercase characters have different values from the lowercase
+letters. This means that 'A' would not be the same as 'a' in Java!
```

Comment:
Perhaps emphasize why this could be important for developers. (Sorting Strings alphabetically might be an example)

Reply:
good idea!